sublime_linter.py

```diff
@@ -398,31 +383,27 @@ def open_tooltip(self, active_view=None, point=None, is_inline=False):
             return
 
         if point:  # provided from hover
-            lineno, colno = active_view.rowcol(point)
+            line, col = active_view.rowcol(point)
         else:
-            lineno, colno = self.get_line_and_col(active_view)
+            line, col = self.get_line_and_col(active_view)
 
-        vid = active_view.id()
-
-        line_dict = persist.errors.get_line_dict(vid, lineno)
-        if not line_dict:
+        bid = active_view.buffer_id()
+        errors = persist.raw_errors[bid]
+        if not errors:
             return
 
-        if is_inline:  # do not show tooltip on hovering empty gutter
-            line_dict = persist.errors.get_region_dict(vid, lineno, colno)
-            show_count = False
-        else:
-            show_count = True
+        if not is_gutter:  # do not show tooltip on hovering empty gutter
+            errors = [e for e in errors if e["start"] <= col <= e["end"]]
 
-        if not line_dict:
+        if not errors:
             return
 
-        we_count = persist.errors.get_we_count_line(vid, lineno)
+        we_count =  # where to get those from?
```

Comment:
That's actually relatively simple. But I go to bed 🥃 and start my weekend.

Reply:
You owe it!